verifiable-db/src/cells_tree/api.rs

```diff
@@ -351,16 +325,11 @@ mod tests {
             pi.multiplier_values_digest_point(),
             values_digests.multiplier.to_weierstrass(),
         );
-        // Check individual metadata digest
-        assert_eq!(
-            pi.individual_metadata_digest_point(),
-            metadata_digests.individual.to_weierstrass(),
-        );
-        // Check multiplier metadata digest
-        assert_eq!(
-            pi.multiplier_metadata_digest_point(),
-            metadata_digests.multiplier.to_weierstrass(),
-        );
+        // Check individual counter
+        let multiplier_cnt = F::from_bool(is_multiplier);
+        assert_eq!(pi.individual_counter(), F::ONE - multiplier_cnt);
```

Comment:
I don't understand this. 
We officially only support single table so the multipler columns can be 1 or more, that means this subtraction can underflow.

Then in my mind the total number of columns = `individual_counter + multiplier_counter` so if we want the `individual_columns = total - multiplier_columns` , not one...

Reply:
This is testing a leaf node of a cells tree, so the counter is either 0 or 1, depending on whether the current cell is accumulated as individual or multiplier